bootstrap/src/lib/cache/index.ts

```diff
@@ -27,6 +31,12 @@ export const defaultOptions = () => ({
       ...(env.CACHE_KEY_IGNORED_PROPS || '').split(',').filter((k) => k), // no empty keys
     ],
   },
+  rateLimit: {
+    groupMaxAge: parseInt(env.GROUP_MAX_AGE || '') || DEFAULT_GROUP_MAX_AGE,
+    groupId: (env.API_KEY && hash(env.API_KEY)) || '',
```

Comment:
What about for composite adapters that use a prefix on their API_KEY? 
Currently Proof of Reserves and Outlier Detection do this.

I see a couple solutions:
1. Have them set API_KEY too (no more work but bad UX)
2. Pass through a prefix to `expose`
3. Set an optional prefix ENV VAR (is this even better UX?, maybe more intuitive naming)

This is kind of a weird case because I don't think we even currently have code that uses multiple API keys.

Reply:
Also makes me think about this case in a general sense - 

We have a composite adapter called price-compare that uses 2 adapters: `CMC` and `coingecko`
Which one do we key off of? `CMC`? It has no way to handle both.
Now what if the same node operator is also using `CMC` as a stand-alone adapter?

This may be an edge case that we can note and avoid until V2.
Or means that individual request caching should be thought of now.

Reply:
Really good point, thanks for bringing it out. Working on it